diff -rupN gamin-0.1.10.orig/server/gam_node.c gamin-0.1.10/server/gam_node.c
--- gamin-0.1.10.orig/server/gam_node.c	2017-11-18 16:50:18.869698771 +0100
+++ gamin-0.1.10/server/gam_node.c	2017-11-18 16:59:50.619339592 +0100
@@ -122,7 +122,7 @@ gam_node_set_is_dir(GamNode * node, gboo
  * it has finished with the string.  If it must keep it longer, it
  * should makes its own copy.  The returned string must not be freed.
  */
-G_CONST_RETURN char *
+const char *
 gam_node_get_path(GamNode * node)
 {
     g_assert(node);
diff -rupN gamin-0.1.10.orig/server/gam_node.h gamin-0.1.10/server/gam_node.h
--- gamin-0.1.10.orig/server/gam_node.h	2017-11-18 16:50:18.869698771 +0100
+++ gamin-0.1.10/server/gam_node.h	2017-11-18 16:59:49.787347510 +0100
@@ -58,7 +58,7 @@ gboolean              gam_node_is_dir
 void                  gam_node_set_is_dir          (GamNode         *node,
 						   gboolean        is_dir);
 	
-G_CONST_RETURN char  *gam_node_get_path            (GamNode         *node);
+const char  *gam_node_get_path            (GamNode         *node);
 
 GList                *gam_node_get_subscriptions   (GamNode         *node);
 
diff -rupN gamin-0.1.10.orig/server/gam_subscription.c gamin-0.1.10/server/gam_subscription.c
--- gamin-0.1.10.orig/server/gam_subscription.c	2017-11-18 16:50:18.869698771 +0100
+++ gamin-0.1.10/server/gam_subscription.c	2017-11-18 16:59:51.447331714 +0100
@@ -141,7 +141,7 @@ gam_subscription_pathlen(GamSubscription
  * @param sub the GamSubscription
  * @returns The path being monitored.  It should not be freed.
  */
-G_CONST_RETURN char *
+const char *
 gam_subscription_get_path(GamSubscription * sub)
 {
     if (sub == NULL)
diff -rupN gamin-0.1.10.orig/server/gam_subscription.h gamin-0.1.10/server/gam_subscription.h
--- gamin-0.1.10.orig/server/gam_subscription.h	2017-11-18 16:50:18.869698771 +0100
+++ gamin-0.1.10/server/gam_subscription.h	2017-11-18 16:59:48.563359165 +0100
@@ -21,7 +21,7 @@ int                  gam_subscription_pa
 
 int                  gam_subscription_get_reqno    (GamSubscription *sub);
 
-G_CONST_RETURN char *gam_subscription_get_path     (GamSubscription *sub);
+const char *gam_subscription_get_path     (GamSubscription *sub);
 
 GamListener         *gam_subscription_get_listener (GamSubscription *sub);
 
