diff -ru perl-5.26.1.orig/lib/perlbug.t perl-5.26.1/lib/perlbug.t
--- perl-5.26.1.orig/lib/perlbug.t	2018-03-24 01:45:27.602832095 +0000
+++ perl-5.26.1/lib/perlbug.t	2018-03-24 01:47:37.312627944 +0000
@@ -148,7 +148,7 @@
 my $maxlen2 = 0; # attachment
 for (split(/\n/, $contents)) {
         my $len = length;
-        $maxlen1 = $len if $len > $maxlen1 and !/$B/;
+        $maxlen1 = $len if $len > $maxlen1 and ! (/(?:$B|PATH)/);
         $maxlen2 = $len if $len > $maxlen2 and  /$B/;
 }
 ok($maxlen1 < 1000, "[perl #128020] long body lines are wrapped: maxlen $maxlen1");
