diff --git a/mongodb/SConstruct b/mongodb/SConstruct
index 4cacc21..3f10000 100644
--- a/mongodb/SConstruct
+++ b/mongodb/SConstruct
@@ -367,6 +367,11 @@ add_option('disable-warnings-as-errors',
     nargs=0,
 )
 
+add_option('propagate-shell-environment',
+           help="Pass shell environment to sub-processes (NEVER for production builds)",
+           nargs=0,
+           default=False)
+
 add_option('detect-odr-violations',
     help="Have the linker try to detect ODR violations, if supported",
     nargs=0,
@@ -622,11 +627,6 @@ env_vars.Add('CXXFLAGS',
     help='Sets flags for the C++ compiler',
     converter=variable_shlex_converter)
 
-# Note: This probably is only really meaningful when configured via a variables file. It will
-# also override whatever the SCons platform defaults would be.
-env_vars.Add('ENV',
-    help='Sets the environment for subprocesses')
-
 env_vars.Add('HOST_ARCH',
     help='Sets the native architecture of the compiler',
     converter=variable_arch_converter,
@@ -1297,6 +1297,9 @@ if link_model.startswith("dynamic"):
 if optBuild:
     env.SetConfigHeaderDefine("MONGO_CONFIG_OPTIMIZED_BUILD")
 
+if has_option("propagate-shell-environment"):
+    env['ENV'] = dict(os.environ);
+
 # Enable the fast decider if exlicltly requested or if in 'auto' mode and not in conflict with other
 # options.
 if get_option('build-fast-and-loose') == 'on' or \
