--- gdb/nat/amd64-linux-siginfo.c       2022-02-04 07:31:57.282699313 +0000
+++ gdb/nat/amd64-linux-siginfo-fix.c   2022-02-04 07:31:09.311533018 +0000
@@ -17,8 +17,8 @@
    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.  */
 
-#include <signal.h>
 #include "common/common-defs.h"
+#include <signal.h>
 #include "amd64-linux-siginfo.h"
 
 #define GDB_SI_SIZE 128
