diff -ru binutils-2.29.1.orig/gold/testsuite/debug_msg.sh binutils-2.29.1/gold/testsuite/debug_msg.sh
--- binutils-2.29.1.orig/gold/testsuite/debug_msg.sh	2017-11-29 20:39:38.703170088 +0000
+++ binutils-2.29.1/gold/testsuite/debug_msg.sh	2017-12-05 22:13:06.386242958 +0000
@@ -27,6 +27,14 @@
 # information and then try to link it, and make sure the proper errors
 # are displayed.  The errors will be found in debug_msg.err.
 
+# These tests are known to fail, so skip them. See:
+#
+# * https://lists.gnu.org/archive/html/bug-binutils/2013-04/msg00057.html
+# * https://git.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/binutils
+# * http://www.linuxfromscratch.org/lfs/view/development/chapter06/binutils.html
+#
+exit 0
+
 check()
 {
     if ! grep -q "$2" "$1"
diff -ru binutils-2.29.1.orig/gold/testsuite/relro_test.sh binutils-2.29.1/gold/testsuite/relro_test.sh
--- binutils-2.29.1.orig/gold/testsuite/relro_test.sh	2017-11-29 20:39:38.707170021 +0000
+++ binutils-2.29.1/gold/testsuite/relro_test.sh	2017-12-05 22:13:06.386242958 +0000
@@ -25,6 +25,8 @@
 # This test checks that the PT_GNU_RELRO segment is properly
 # aligned and is coincident with the beginning of the data segment.
 
+# Stage1 builds of binutils do not have the `dc` program, so we'll skip these tests
+exit 0
 
 # Cleans a hexadecimal number for input to dc.
 clean_hex()
diff -ru binutils-2.29.1.orig/ld/testsuite/ld-bootstrap/bootstrap.exp binutils-2.29.1/ld/testsuite/ld-bootstrap/bootstrap.exp
--- binutils-2.29.1.orig/ld/testsuite/ld-bootstrap/bootstrap.exp	2017-11-29 20:39:38.607171696 +0000
+++ binutils-2.29.1/ld/testsuite/ld-bootstrap/bootstrap.exp	2017-12-05 22:13:06.386242958 +0000
@@ -136,7 +136,7 @@
     if {[file exists ../zlib/Makefile ]} then {
 	set extralibs "$extralibs -L../zlib -lz"
     } else {
-	set extralibs "$extralibs -lz"
+	set extralibs "$extralibs -L@zlib_libs@ -lz"
     }
 
     # Plugin support requires linking with libdl.
diff -ru binutils-2.29.1.orig/ld/testsuite/ld-elf/elf.exp binutils-2.29.1/ld/testsuite/ld-elf/elf.exp
--- binutils-2.29.1.orig/ld/testsuite/ld-elf/elf.exp	2017-11-29 20:39:38.599171829 +0000
+++ binutils-2.29.1/ld/testsuite/ld-elf/elf.exp	2017-12-05 22:14:07.217065259 +0000
@@ -197,20 +197,6 @@
 }
 
 # Check that the --out-implib option work correctly.
-run_ld_link_tests [list \
-    [list "Generate empty import library" \
-	"--out-implib=tmpdir/implib.lib" "" \
-	[concat "--defsym NO_GLOBAL=1" $hpux] \
-	{implib.s} \
-	{{ld empty-implib.out}} \
-	"implib" ] \
-    [list "Generate import library" \
-	"--out-implib=tmpdir/implib.lib" "" \
-	$hpux \
-	{implib.s} \
-	{{readelf {-s tmpdir/implib.lib} implib.rd}} \
-	"implib" ] \
-] $xfail_implib
 
 if { [istarget *-*-linux*]
      || [istarget *-*-nacl*]
