diff -Naur a/makedefs b/makedefs
--- a/makedefs	2023-08-22 11:01:14.461101408 +0000
+++ b/makedefs	2023-08-22 11:04:13.001023406 +0000
@@ -568,7 +568,7 @@
 		: ${SHLIB_ENV="LD_LIBRARY_PATH=`pwd`/lib"}
 		: ${PLUGIN_LD="${CC-gcc} -shared"}
 		;;
- Linux.[345].*)	SYSTYPE=LINUX$RELEASE_MAJOR
+ Linux.[3456].*)	SYSTYPE=LINUX$RELEASE_MAJOR
 		case "$CCARGS" in
 		 *-DNO_DB*) ;;
 		 *-DHAS_DB*) ;;
diff -Naur a/src/util/sys_defs.h b/src/util/sys_defs.h
--- a/src/util/sys_defs.h	2023-08-22 11:01:14.537101381 +0000
+++ b/src/util/sys_defs.h	2023-08-22 11:07:20.316948827 +0000
@@ -749,7 +749,7 @@
  /*
   * LINUX.
   */
-#if defined(LINUX2) || defined(LINUX3) || defined(LINUX4) || defined(LINUX5)
+#if defined(LINUX2) || defined(LINUX3) || defined(LINUX4) || defined(LINUX5) || defined(LINUX6)
 #define SUPPORTED
 #define UINT32_TYPE	unsigned int
 #define UINT16_TYPE	unsigned short
