diff -ru mg-20171014.orig/GNUmakefile mg-20171014/GNUmakefile
--- mg-20171014.orig/GNUmakefile	2018-03-01 19:58:39.118807329 +0000
+++ mg-20171014/GNUmakefile	2018-03-01 20:00:16.377297903 +0000
@@ -7,15 +7,15 @@
 
 name=		mg
 
-prefix=		/usr/local
+prefix=		@prefix@
 bindir=		$(prefix)/bin
 libdir=		$(prefix)/lib
 includedir=	$(prefix)/include
 mandir=		$(prefix)/man
 
-PKG_CONFIG=	/usr/bin/pkg-config --silence-errors
-INSTALL=	/usr/bin/install
-STRIP=		/usr/bin/strip
+PKG_CONFIG=	pkg-config --silence-errors
+INSTALL=	install
+STRIP=		strip
 
 UNAME:=		$(shell uname)
 ifeq ($(UNAME),FreeBSD)
diff -ru mg-20171014.orig/Makefile mg-20171014/Makefile
--- mg-20171014.orig/Makefile	2018-03-01 19:58:39.118807329 +0000
+++ mg-20171014/Makefile	2018-03-01 20:02:25.327299700 +0000
@@ -12,6 +12,9 @@
 #
 CFLAGS+=-Wall -DREGEX
 
+CFLAGS += -I@clens_prefix@/include/clens -isystem @libbsd_prefix@/include/bsd -DLIBBSD_OVERLAY -D__dead -O2
+LDADD+=<------>-lcurses -lutil -lclens -lbsd
+
 SRCS=	autoexec.c basic.c bell.c buffer.c cinfo.c dir.c display.c \
 	echo.c extend.c file.c fileio.c funmap.c help.c kbd.c keymap.c \
 	line.c macro.c main.c match.c modes.c paragraph.c \
