--- libcudf/tests/catch.hpp	2021-01-11 12:57:03.000000000 +0000
+++ libcudf/tests/MINSIGSTKSZ_catch.hpp	2022-01-31 07:02:54.676602562 +0000
@@ -10819,7 +10819,9 @@
 
     // 32kb for the alternate stack seems to be sufficient. However, this value
     // is experimentally determined, so that's not guaranteed.
-    static constexpr std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
+    //static constexpr std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
+    // Update: MINSIGSTKSZ is not const anymore with recent glibc
+    static constexpr std::size_t sigStackSize = 32768;
 
     static SignalDefs signalDefs[] = {
         { SIGINT,  "SIGINT - Terminal interrupt signal" },
