--- SConstruct.orig	2016-11-08 13:17:10.000000000 +0000
+++ SConstruct	2019-04-25 17:13:47.215423000 +0000
@@ -143,7 +143,11 @@
 # Set up and export default build environment
 #
 
-env = Environment(ENV = {'PATH' : os.environ['PATH'], 'HOME' : os.environ['HOME']})
+env = Environment(ENV = { 
+  'PATH' : os.environ['PATH'], 
+  'HOME' : os.environ['HOME'], 
+  'LD_RUN_PATH' : os.environ['LD_RUN_PATH']
+})
 
 # Set up environment for ccache and distcc
 #env['ENV']['HOME']          = os.environ['HOME']
