--- hack/lib/golang.sh	2018-04-14 14:13:55.209257922 +0000
+++ hack/lib/golang.sh	2018-04-14 14:15:05.272683732 +0000
@@ -339,7 +339,7 @@
   go_version=($(go version))
   local minimum_go_version
   minimum_go_version=go1.8.3
-  if [[ "${go_version[2]}" < "${minimum_go_version}" && "${go_version[2]}" != "devel" ]]; then
+  if [[ "${minimum_go_version}" != $(echo -e "${minimum_go_version}\n${go_version[2]}" | sort -s -t. -k 1,1 -k 2,2n -k 3,3n | head -n1) && "${go_version[2]}" != "devel" ]]; then
     kube::log::usage_from_stdin <<EOF
 Detected go version: ${go_version[*]}.
 Kubernetes requires ${minimum_go_version} or greater.
