diff -ur jbigkit-2.1-orig/Makefile jbigkit-2.1/Makefile
--- jbigkit-2.1-orig/Makefile	2016-07-17 18:04:03.253877290 +0000
+++ jbigkit-2.1/Makefile	2016-07-17 18:20:23.224271207 +0000
@@ -2,15 +2,13 @@
 
 # Select an ANSI/ISO C compiler here, GNU gcc is recommended
 CC = gcc
-
-# Options for the compiler: A high optimization level is suggested
-CFLAGS = -O2 -W -Wno-unused-result
-# CFLAGS = -O -g -W -Wall -Wno-unused-result -ansi -pedantic # -DDEBUG
-
-export CC CFLAGS
+CFLAGS += -g -O -W -Wall -ansi -pedantic -fPIC -DPIC
+CPPFLAGS += -I../libjbig
 
 VERSION=2.1
 
+export VERSION CC CFLAGS CPPFLAGS
+
 all: lib pbm
 	@echo "Enter 'make test' in order to start some automatic tests."
 
@@ -22,7 +20,7 @@
 
 test: lib pbm
 	cd libjbig  && $(MAKE) test
-	cd pbmtools && $(MAKE) test
+	cd pbmtools && LD_LIBRARY_PATH=../libjbig $(MAKE) test
 
 analyze:
 	cd libjbig  && $(MAKE) analyze
diff -ur jbigkit-2.1-orig/libjbig/Makefile jbigkit-2.1/libjbig/Makefile
--- jbigkit-2.1-orig/libjbig/Makefile	2016-07-17 17:15:07.663497199 +0000
+++ jbigkit-2.1/libjbig/Makefile	2016-07-17 18:19:39.654503192 +0000
@@ -4,25 +4,24 @@
 CC = gcc
 
 # Options for the compiler: A high optimization level is suggested
-CFLAGS = -g -O -W -Wall -ansi -pedantic # --coverage
+CFLAGS += -g -O -W -Wall -ansi -pedantic -fPIC -DPIC
 
-all: libjbig.a libjbig85.a tstcodec tstcodec85
+all: libjbig.so libjbig85.so tstcodec tstcodec85
 
-tstcodec: tstcodec.o jbig.o jbig_ar.o
-	$(CC) $(CFLAGS) -o tstcodec tstcodec.o jbig.o jbig_ar.o
+tstcodec: tstcodec.o
+	$(CC) $(CFLAGS) -o tstcodec tstcodec.o -L. -ljbig
 
-tstcodec85: tstcodec85.o jbig85.o jbig_ar.o
-	$(CC) $(CFLAGS) -o tstcodec85 tstcodec85.o jbig85.o jbig_ar.o
+tstcodec85: tstcodec85.o
+	$(CC) $(CFLAGS) -o tstcodec85 tstcodec85.o -L. -ljbig85
 
-libjbig.a: jbig.o jbig_ar.o
-	rm -f libjbig.a
-	ar rc libjbig.a jbig.o jbig_ar.o
-	-ranlib libjbig.a
+%.so: %.so.$(VERSION)
+	ln -sf $< $@
 
-libjbig85.a: jbig85.o jbig_ar.o
-	rm -f libjbig85.a
-	ar rc libjbig85.a jbig85.o jbig_ar.o
-	-ranlib libjbig85.a
+libjbig.so.$(VERSION): jbig.o jbig_ar.o
+	$(CC) $(CFLAGS) -shared -Wl,-soname,$@ -o $@ $^
+
+libjbig85.so.$(VERSION): jbig85.o jbig_ar.o
+	$(CC) $(CFLAGS) -shared -Wl,-soname,$@ -o $@ $^
 
 jbig.o: jbig.c jbig.h jbig_ar.h
 jbig85.o: jbig85.c jbig85.h jbig_ar.h
@@ -43,13 +42,13 @@
 	clang --analyze *.c
 
 test: tstcodec tstcodec85
-	./tstcodec
-	./tstcodec85
+	LD_LIBRARY_PATH=$(PWD) ./tstcodec
+	LD_LIBRARY_PATH=$(PWD) ./tstcodec85
 
 t82test.pbm: tstcodec
 	./tstcodec $@
 
 clean:
-	rm -f *.o *.gcda *.gcno *.gcov *.plist *~ core gmon.out dbg_d\=??.pbm
+	rm -f *.so *.so.* *.o *.gcda *.gcno *.gcov *.plist *~ core gmon.out dbg_d\=??.pbm
 	rm -f t82test.pbm
 	rm -f tstcodec tstcodec85
diff -ur jbigkit-2.1-orig/pbmtools/Makefile jbigkit-2.1/pbmtools/Makefile
--- jbigkit-2.1-orig/pbmtools/Makefile	2016-07-17 17:15:07.663497199 +0000
+++ jbigkit-2.1/pbmtools/Makefile	2016-07-17 18:19:39.654503192 +0000
@@ -4,8 +4,8 @@
 CC = gcc
 
 # Options for the compiler
-CFLAGS = -g -O -W -Wall -Wno-unused-result -ansi -pedantic # --coverage
-CPPFLAGS = -I../libjbig 
+CFLAGS += -g -O -W -Wall -Wno-unused-result -ansi -pedantic # --coverage
+CPPFLAGS += -I../libjbig 
 
 .SUFFIXES: .1 .5 .txt $(SUFFIXES)
 .PHONY: txt test test82 test85 clean
@@ -14,16 +14,16 @@
 
 txt: pbmtojbg.txt jbgtopbm.txt pbm.txt pgm.txt
 
-pbmtojbg: pbmtojbg.o ../libjbig/libjbig.a
+pbmtojbg: pbmtojbg.o ../libjbig/libjbig.so
 	$(CC) $(CFLAGS) -o pbmtojbg pbmtojbg.o -L../libjbig -ljbig
 
-jbgtopbm: jbgtopbm.o ../libjbig/libjbig.a
+jbgtopbm: jbgtopbm.o ../libjbig/libjbig.so
 	$(CC) $(CFLAGS) -o jbgtopbm jbgtopbm.o -L../libjbig -ljbig
 
-pbmtojbg85: pbmtojbg85.o ../libjbig/libjbig85.a
+pbmtojbg85: pbmtojbg85.o ../libjbig/libjbig85.so
 	$(CC) $(CFLAGS) -o pbmtojbg85 pbmtojbg85.o -L../libjbig -ljbig85
 
-jbgtopbm85: jbgtopbm85.o ../libjbig/libjbig85.a
+jbgtopbm85: jbgtopbm85.o ../libjbig/libjbig85.so
 	$(CC) $(CFLAGS) -o jbgtopbm85 jbgtopbm85.o -L../libjbig -ljbig85
 
 jbgtopbm.o: jbgtopbm.c ../libjbig/jbig.h
@@ -31,13 +31,13 @@
 jbgtopbm85.o: jbgtopbm85.c ../libjbig/jbig85.h
 pbmtojbg85.o: pbmtojbg85.c ../libjbig/jbig85.h
 
-../libjbig/libjbig.a: ../libjbig/jbig.c ../libjbig/jbig.h \
+../libjbig/libjbig.so: ../libjbig/jbig.c ../libjbig/jbig.h \
 	../libjbig/jbig_ar.c ../libjbig/jbig_ar.h
-	make -C ../libjbig libjbig.a
+	make -C ../libjbig libjbig.so
 
-../libjbig/libjbig85.a: ../libjbig/jbig85.c ../libjbig/jbig85.h \
+../libjbig/libjbig85.so: ../libjbig/jbig85.c ../libjbig/jbig85.h \
 	../libjbig/jbig_ar.c ../libjbig/jbig_ar.h
-	make -C ../libjbig libjbig85.a
+	make -C ../libjbig libjbig85.so
 
 analyze:
 	clang $(CPPFLAGS) --analyze *.c
